notebooks/integration/interactive_units.ipynb

```diff
@@ -0,0 +1,161 @@
+{
```

Comment:
An explanation in the notebook would be nice.

Reply:
I added some text in the beginning of the notebook!